sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -67,7 +67,11 @@ private[hive] object IsolatedClientLoader extends Logging {
           case e: RuntimeException if e.getMessage.contains("hadoop") =>
             // If the error message contains hadoop, it is probably because the hadoop
             // version cannot be resolved.
-            val fallbackVersion = "2.7.4"
+            val fallbackVersion = VersionUtils.majorMinorPatchVersion(hadoopVersion) match {
```

Comment:
maybe we can use `VersionUtils.isHadoop3` here.

Reply:
How about current